decidim-admin/app/views/decidim/admin/participatory_process_attachments/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h3><%= translated_attribute(@participatory_process_attachment.title) %></h3>
```

Comment:
Don't think so. Anyway, we don't have enough information, let's wait until we have the design. It's just that they kinda add bloat.